README.md

```diff
@@ -1,2 +1,24 @@
 # hackathon-2021-stateful-flink-processes
-Resources to making Flink processes stateful
+Resources for making Flink processes stateful
+
+# Releasing the docker image
+cp -R <FLINK_DISTRIBUTION_PATH>/* flink/ 
+docker build --tag 281050902442.dkr.ecr.eu-central-1.amazonaws.com/hackathon-2021-stateful-flink-process/flink:1.15-SNAPSHOT --file Dockerfile .
+docker push 281050902442.dkr.ecr.eu-central-1.amazonaws.com/hackathon-2021-stateful-flink-process/flink:1.15-SNAPSHOT
```

Comment:
We probably should create a new tag whenever we create a new image, shouldn't we? Then we don't have to always pull images on the nodes.

Reply:
Yes, exactly.

Reply:
Updated.